sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -125,13 +125,13 @@ CREATE UNIQUE INDEX `uuid` ON `insurance_companies` (`uuid`);
 ALTER TABLE `insurance_data` ADD `uuid` binary(16) DEFAULT NULL;
```

Comment:
at the top of this script, lets add the following (will plan to do this on all upgrades until no longer need to; when uuid is built into all of openemr's queries):
```
#IfUuidNeedUpdate patient_data
#EndIf

#IfUuidNeedUpdate form_encounter
#EndIf

#IfUuidNeedUpdate users
#EndIf

#IfUuidNeedUpdateVertical facility_user_ids uid:facility_id
#EndIf

#IfUuidNeedUpdate facility
#EndIf

#IfUuidNeedUpdate immunizations
#EndIf

#IfUuidNeedUpdate lists
#EndIf

#IfUuidNeedUpdateId procedure_order procedure_order_id
#EndIf

#IfUuidNeedUpdateId drugs drug_id
#EndIf

#IfUuidNeedUpdate prescriptions
#EndIf

#IfUuidNeedUpdateId procedure_result procedure_result_id
#EndIf

#IfUuidNeedUpdate ccda
#EndIf
```